webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const EmptyState = (props) => {
+  const {
+    icon = 'add-circle-o',
+    header,
+    description,
+    customDocumentation,
+    documentationLabel = 'For more information please see',
+    documentationButton = 'Documentation',
+    docUrl,
```

Comment:
I still would prefer to see this as an object like the action.  We should be able to customize the text of this button instead of it always being "Documentation".